File: workspace/pset1/mario/less/mario.c
// Draws hash steps from left to right. Last step is double as wide as others

#include <stdio.h>
#include <cs50.h>

int main(void)
{
    // Get height by user
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 0 || height > 23);

    // Print half-pyramid
    char c;
    for (int i = 0; i < height; i++)
    {
        for (int j = 0; j <= height; j++)
        {
            c = (j < height - 1 - i) ? ' ' : '#';
            printf("%c", c);
        }
        printf("\n");
    }
}

File: workspace/pset6/mario/less/mario.py
# Draws hash steps from left to right. Last step is double as wide as others

from cs50 import get_int

# Gets height by user
height = -1
while height < 0 or height > 23:
    height = get_int("Height: ")

# Prints half pyramid
# Iterate rows
for i in range(height):
    # Iterate collumns
    for j in range(height + 1):
        print(" " if j < height - 1 - i else "#", end="")
    print()